tox.ini

```diff
@@ -80,6 +81,22 @@ commands =
     python -c 'from shutil import rmtree; rmtree("build", True); rmtree("dist", True)'
     python -m build .
 
+[syftcli.build]
+basepython = python3
+changedir = {toxinidir}/packages/syftcli
+description = Build SyftCLI Binary for each platform
+commands =
+    python -m pip install --upgrade pip
+    pip install --upgrade setuptools wheel twine tox build
+    pip install -e ".[build]"
+    python -c 'from shutil import rmtree; rmtree("build", True); rmtree("dist", True)'
+    ;check if we are on macos add additional flags if not run as is
+    bash -c 'if [[ "$OSTYPE" == "darwin"* ]]; then \
+         pyinstaller --clean --onefile --name syftcli --distpath ./dist/cli src/syftcli/cli.py --target-arch universal2; \
```

Comment:
MAYBE: By default, the distpath , takes ./dist, do we want to place it in ./dist/cli/src/syftcli/cli.py ?

We could keep dist itself?

Reply:
Q: We should provide the path to cli.py ?

like

```python3
pyinstaller --clean --onefile --name syftcli  src/syftcli/cli.py --target-arch universal2;
```

?

Reply:
The distpath is actually definitive. When that runs, it will still place it in ./dist/cli not ./dist/cli/src/syftcli/cli.py. 

Also for Q2, we specified the path to cli.py , after the distpath argument. so it's more like `pyinstaller --clean --onefile --name  --distpath src`